### ISSUE TITLE
Importar cartas periodicamente

### ISSUE THREAD

**Issue by cv** [servicosgovbr/portal-de-servicos#406]
Agora que movemos as cartas para outro repositório (#399), podemos desacoplar deployments da aplicação a importações das cartas.

Periodicamente (12 em 12h, talvez?), descobrir qual a última revisão "verde" das cartas no Snap - precisamos garantir que os testes necessários estão sendo executados! - e importá-las, preferencialmente sem downtime no Portal.

**Comment by sahrosa** [comment]
Isso fica nesse repositório mesmo, @cv?

**Comment by cv** [comment]
Sim, a importação de cartas periódica tem que acontecer no portal. No editor, atualizamos sempre que carregamos um arquivo.

**Comment by cv** [comment]
Este cartão se torna prioridade no Portal assim que tivermos o https://github.com/servicosgovbr/editor-de-servicos/issues/282 implementado. Depende do #443, também.

**Comment by cv** [comment]
Adicionei a importação da versão `master` das cartas a cada minuto múltiplo de 10 (8:30, 8:40, 8:50...) de segunda a sexta-feira.

Pode ser que tenhamos algumas surpresas caso a importação seja interrompida por um novo deploy ou outra operação que destrua a JVM durante a execução.

O risco disso causar problemas se acontecer é bem baixo, e no pior dos casos vai se resolver sozinho 10 minutos depois.

**Comment by cv** [comment]
Para validar: http://bit.ly/1JDVhIL
